5.5/Dockerfile.rhel7

```diff
@@ -20,13 +20,13 @@ LABEL BZComponent="openshift-sti-php-docker" \
       Release="1" \
       Architecture="x86_64"
 
-# Install Apache httpd and PHP
-RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+# Install subscription-manager, Apache httpd and PHP
+RUN yum install -y subscription-manager && \
```

Comment:
Weird, we don't install `subscription-manager` anywhere else either. It should be enough to have your RHEL-7 host properly subscribed; is that the case? I don't think the Dockerfile can be built anywhere else than on subscribed RHEL-7 host, at least I'm not aware of anything.